modules/core/include/opencv2/core/hal/intrin.hpp

```diff
@@ -1226,6 +1226,105 @@ namespace CV__SIMD_NAMESPACE {
 //! @endcond
 
 
+//! @name Exponential
+//! @{
+//! @brief Calculates exponent of every vector element
```

Comment:
Almost all documentation in this file is ignored.

This file contains too much conditional compilations through `#if` and template-based logic.
Doxygen doesn't work properly with that.

Documentation is stored and well structured in intrin_cpp.hpp

Reply:
OK, I added a line in `intrin_cpp.hpp` for document
```
| Operations\\Types | float 32 | float 64 |
|-------------------|:-:|:-:|
|exp                | x | x |
```